common/trinary/trit_array.c

```diff
@@ -113,6 +113,29 @@ trit_array_p trit_array_new_from_trytes(tryte_t const *const trytes) {
   return trit_array;
 }
 
+trit_array_p trit_array_new_from_flex(flex_trit_t const *const trits,
```

Comment:
Not sure this function is really needed following my comments

Reply:
No needed, if `iota_tangle_transaction_load` accept flex_trit.

Reply:
Understood, but what I meant was that we could achieve the exact same thing with no dynamic allocation and no memcpy.